src/app/app.component.ts

```diff
@@ -375,4 +345,61 @@ export class AppComponent {
   private shouldShowChat(eventUrl: string): boolean {
     return (this._app.config.showZendeskWidget && this.validChatUrls.includes(eventUrl));
   }
+
+  private loadWidgetScripts(withOneTrust: boolean) {
```

Comment:
Would rename to `initializeWidgets` based on the current code organization.

Reply:
Changes made in `b6710cb`.